bindings/rust/build.rs

```diff
@@ -4,16 +4,16 @@ fn main() {
     let mut c_config = cc::Build::new();
     c_config.std("c11").include(src_dir);
 
+    #[cfg(target_env = "msvc")]
+    c_config.flag("-utf-8");
+
     let parser_path = src_dir.join("parser.c");
     c_config.file(&parser_path);
     println!("cargo:rerun-if-changed={}", parser_path.to_str().unwrap());
 
-    // NOTE: if your language uses an external scanner, uncomment this block:
-    /*
-    let scanner_path = src_dir.join("scanner.c");
-    c_config.file(&scanner_path);
-    println!("cargo:rerun-if-changed={}", scanner_path.to_str().unwrap());
-    */
+    // let scanner_path = src_dir.join("scanner.c");
```

Comment:
Please delete the commented code if deleting the comment explaining why it is there

Reply:
I tried a scanner for better comments parsing but was not worth the effort for that simple kind of comments, defined for WIT.
Should be reverted to what it was before. Still some commented code (with explanation) for easy enablement of scanners.